app/views/disbursements/_form.html.erb

```diff
@@ -1,57 +1,10 @@
 <% disabled = @source_event.present? && !policy(@source_event).create_transfer? %>
 
-<%= form_with(model: disbursement, local: true, url: disbursements_path, html: { "x-data" => "{ fee: null, event_id: null }" }) do |form| %>
+<%= form_with(model: disbursement, local: true, url: disbursements_path, html: { "x-data" => "{fee: null, event_id: null}", "data-turbo" => "false" }) do |form| %>
```

Comment:
Any reason for removing Turbo?

Reply:
This is how it was previously (before the slug change) and it worked fine. Is there any reasoning for adding it back in?

Reply:
It was removed in https://github.com/hackclub/hcb/commit/5d357fa269a96e679c9aaa3416f7c706801d9bd2#diff-1c5acab3fa3dfad661adbde557e22b311bde4e7fb1dadafaf85dd5d2f0861edaR3-L9, before then we'd been using Turbo on this form. I think we should keep Turbo for consistency across all forms.

Reply:
Ah got it. I think there was a reason I had removed it, but I'll try testing it with Turbo to see if we can keep it.